CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.9.3](https://github.com/rudderlabs/rudder-server/compare/v1.9.2...v1.9.3) (2023-05-31)
+
+
+### Bug Fixes
+
+* **batchrouter:** batchrouter stops processing events for destinations where a destType-specific config option is set ([#3421](https://github.com/rudderlabs/rudder-server/issues/3421)) ([f2dc53c](https://github.com/rudderlabs/rudder-server/commit/f2dc53cf6a976c09c6c9f45b0b3d846fbcd55745))
+* **batchrouter:** batchrouter stops processing events when BatchRouter.&lt;DestType&gt; configuration option is set ([f2dc53c](https://github.com/rudderlabs/rudder-server/commit/f2dc53cf6a976c09c6c9f45b0b3d846fbcd55745))
+
```

Comment:
@atzoum Should this be present in Changelog?

Reply:
fixed